Makefile

```diff
@@ -22,6 +22,7 @@ lint:
 	golint `go list ./... | grep -v /vendor/`
 
 build:
+	swag init -g core/api.go
```

Comment:
Now it'll automatically rebuild docs, which doesn't take a ton of extra time, but we _might_ want to think about having this in a separate make target?

Reply:
seems like a good idea to have it separate, build is for the binary

Reply:
Agreed, moved